keepalive/keepalive.go

```diff
@@ -33,7 +33,19 @@ import (
 type ClientParameters struct {
 	// After a duration of this time if the client doesn't see any activity it
 	// pings the server to see if the transport is still alive.
-	// If set below 10s, a minimum value of 10s will be used instead.
+	// If set below 10s (including 0), a minimum value of 10s will be used
+	// instead.
+	//
+	// Note that gRPC servers have a default EnforcementPolicy.MinTime of 5
+	// minutes (which means the client shouldn't ping more frequently than every
+	// 5 minutes).
+	//
+	// Though not ideal, it's not a strong requirement for Time to be less than
+	// EnforcementPolicy.MinTime.  Time will automatically double if the server
+	// disconnects due to its enforcement policy.
+	//
+	// For more details, see
+	// https://github.com/grpc/proposal/blob/master/A8-client-side-keepalive.md
 	Time time.Duration // The current default value is infinity.
```

Comment:
Can you delete this comment, please?  Explaining the behavior can be done via the existing text above and in `grpc.WithKeepaliveParams` (which ideally would be updated as part of this cleanup, if you don't mind, to say something like "`Keepalive is disabled by default`" or "`Keepalive pings are disabled by default`").

Reply:
Done